federatedscope/core/configs/cfg_training.py

```diff
@@ -46,6 +46,9 @@ def extend_training_cfg(cfg):
     # when model is too large, users can use half-precision model
     cfg.train.is_enable_half = False
 
+    # check whether model uses accelerator
+    cfg.train.use_accelerator = False
```

Comment:
what is this args used for?

Reply:
I have implemented additional packages to be used in the application of differential privacy in FedScope, where [accelerator](https://github.com/huggingface/accelerate) is useful.